misc/snapshotter/config.toml

```diff
@@ -65,8 +66,8 @@ enable_kubeconfig_keychain = false
 kubeconfig_path = ""
 # Fetch the private registry auth as CRI image service proxy
 enable_cri_keychain = false
-# the target image service when using image proxy
-image_service_address = ""
+# the target image service when using image proxy, default: /run/containerd/containerd.sock
```

Comment:
Does snapshotter really set the default value `/run/containerd/containerd.sock` ?

Reply:
You are right, fixed